src/components/BottomNav.jsx

```diff
@@ -7,7 +7,7 @@ import { setIsAddingTask } from 'redux/actions/TodoAction';
 function BottomNav() {
   const dispatch = useDispatch();
 
-  function setIsAdding(event) {
+  function onAdding(event) {
```

Comment:
May be we can avoid continuous tense for functions. Thanks!
```suggestion
  function onAdd(event) {
```

Reply:
updated bhai